notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<p>"Use of <code>to_timestamp()</code> and <code>unix_timestamp()</code> function..."</p><p>-&gt;</p><p>"Use the <code>to_timestamp()</code> and <code>unix_timestamp()</code> functions"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>